app/web/features/communities/CommunityBase.tsx

```diff
@@ -38,10 +38,9 @@ export default function CommunityBase({
     isLoading: isCommunityLoading,
     error: communityError,
     data: community,
-    queryCommunityId,
-  } = useCommunity(communityId);
+  } = useCommunity(communityId ?? 0);
```

Comment:
Could we not still have the defaulting logic as part of the `useCommunity` hook as before? Too many files changed here in the GitHub diff for me to see whether the implementation of this has been changed 😅